package.json

```diff
@@ -29,17 +29,19 @@
   "homepage": "https://github.com/cyverse-de/sonora#readme",
   "nodemonConfig": {
     "watch": [
-      "server",
-      "common",
-      "pages"
+      "src/server",
+      "src/common"
     ],
     "ext": "js",
     "exec": "run-p dev-server build-docs"
   },
   "eslintConfig": {
     "extends": [
       "react-app"
-    ]
+    ],
+    "rules": {
+       "react/react-in-jsx-scope": "off"
```

Comment:
I guess this is the standard in Next.js React files (I noticed the lack of React imports in their tutorials).
It still feels a bit weird to me, but I suppose I'll get used to it 😅

Reply:
Feels weird to me, too.